src/zope/viewlet/manager.py

```diff
@@ -155,16 +217,23 @@ class WeightOrderedViewletManager(ViewletManagerBase):
     """Weight ordered viewlet managers."""
 
     def sort(self, viewlets):
+        """
+        Sort the viewlets based on their ``weight`` attribute (if present;
+        viewlets without a ``weight`` are sorted at the beginning but are
+        otherwise  unordered).
```

Comment:
Double space.

Reply:
Thank you. I'm using [a new comment formatter](https://github.com/glyph/python-docstring-mode) that's RST and doctest aware, and it usually works great, but it does seem to have an issue of inserting extra spaces sometimes.